Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java

```diff
@@ -12,6 +13,7 @@
 import java.util.Date;
 
 public class ListActivity extends AppCompatActivity {
+  public static Activity activity;
```

Comment:
Same naming issue here. If we can give a name explains it's real function it might be better.
If the name `activity` is doing its job pretty well, than you can keep it.